city_local/pdx/events/workplace-training.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: PDX October 2019 Workplace training
+permalink: /pdx/events/workplace-training
+---
+
+Tech Worker Organizer Training is a six hour class which aims to give people the minimum skills and knowledge to get started safely and responsibly with workplace organizing as tech workers (workers in the tech industry or technical workers in any industry).
```

Comment:
We thought TWOT was a good name lol and also it's the most obvious one, but open to suggestions